deployability/launchers/allocation.py

```diff
@@ -19,6 +19,7 @@ def parse_arguments():
     parser.add_argument("--track-output", required=False, default='/tmp/wazuh-qa/track.yml')
     parser.add_argument("--inventory-output", required=False, default='/tmp/wazuh-qa/inventory.yml')
     parser.add_argument("--working-dir", required=False, default='/tmp/wazuh-qa')
+    parser.add_argument("--label", action='append', required=False, default=None)
```

Comment:
Wouldn't it be better to have a parameter for each label (termination_date, issue, and team) instead of a single one?

Reply:
Done: f8244cea4